config/locales/en.yml

```
@@ -140,6 +140,16 @@ en:
       employed_directly:
         "yes": "Yes, I’m employed by my school"
         "no": "No, I’m employed by a private agency"
+    admin:
+      teaching_maths_or_physics: "Teach any maths or physics?"
```

The other questions here are written in the third person e.g. "Has entire term contract?", so I think this would be more consistent:

```suggestion
      teaching_maths_or_physics: "Teaches any maths or physics?"
```